src/nxdoc/nuxeo-add-ons/nuxeo-coldstorage/nuxeo-coldstorage-functional-overview.md

```diff
@@ -0,0 +1,96 @@
+
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/Nuxeo Coldstorage/Send to Coldstorage Confirmation
+    name: SendToCS2.png
+    addins#screenshot#up_to_date
+--}}
+![Send to Coldstorage Confirmation](nx_asset://2970838e-c5d6-4438-a0cf-eb2fcf4671f9)
+
+## Send multiple documents to Coldstorage
+There's an option to send multiple documents to coldstorage, if all of them satisfy the requirements (they are not yet in coldstorage) and if the user has the wrights. Select the files that you want to upload and there will be an action button to move them to coldstorage.
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/Nuxeo Coldstorage/Send Multiple To Coldstorage
+    name: SendMultipleToCS1.png
+    addins#screenshot#up_to_date
+--}}
+![Send Multiple To Coldstorage](nx_asset://d6f239bd-b830-4b3e-a041-bc23ec2d985c)
+
+## Retrieve from Coldstorage
+When the user wants to have temporary acess to the full file for download, he can trigger the retrieve flow. It is required that the file is already fully stored in nuxeo coldstorage - in other terms, that the document AWS storage class transitioned to glacier.
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/Nuxeo Coldstorage/Restore from coldstorage button
+    name: RestoreFromCS1.png
+    addins#screenshot#up_to_date
+--}}
+![Restore from coldstorage button](nx_asset://bd6ceed7-004f-4d78-a0e7-2740b4871ce0)
+
+The user can then trigger the flow by clicking on the restore button. As an asyncronous process, it is expected that it takes some time. An email is sent to the user when the retrieve process is done and also the document will display a banner stating that the retrieve is in process.
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/Nuxeo Coldstorage/Retrieve from Coldstorage
+    name: RetrieveFromCS2.png
+    addins#screenshot#up_to_date
+--}}
+![Retrieve from Coldstorage](nx_asset://d606f54c-7113-4ce6-a319-be5486c40159)
+
+Once the retrieve is done, the banner will display information about the preview availability and the option to download the content.
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/Nuxeo Coldstorage/Retrieved from Coldstorage 
+    name: RetrieveFromCS3.png
+    addins#screenshot#up_to_date
+--}}
+![Retrieved from Coldstorage ](nx_asset://514f7467-bce2-404c-9d76-0864c1869b57)
+
+
+By default, the file will be available for download for 24h. For now, that value can be changed for all files by adding a configuration on nuxeo-conf:
+
+```
+nuxeo.coldstorage.numberOfDaysOfAvailability.value.default=
```

Comment:
We should specify what the unit here is. Perhaps even give an example. If I want it to be available for 48h, what should we specify? 48 or 2?